src/store/middlewares/login.js

```diff
@@ -17,30 +14,29 @@ const loginMiddleware = store => next => (action) => { // eslint-disable-line ma
           data: { isHarwareWalletConnected: false },
         });
 
-        ipc.on('ledgerConnected', () => {
+        ipc.on('hwConnected', (event, { model }) => {
           store.dispatch({
             type: actionTypes.settingsUpdated,
             data: { isHarwareWalletConnected: true },
           });
-          store.dispatch(errorToastDisplayed({ label: HW_MSG.LEDGER_CONNECTED }));
+          store.dispatch(errorToastDisplayed({ label: `${model} connected` }));
```

Comment:
I would expect green (or neutral) toast here. There is nothing "erroneous" about this event
```suggestion
          store.dispatch(successToastDisplayed({ label: `${model} connected` }));
```

Reply:
I think it should be part of the login process but it doesn't need to be part of middleware and it should happen inside the login action.